docs/src/rules/no-invalid-this.md

```diff
@@ -284,6 +284,63 @@ obj.Foo = function Foo() {
 
 :::
 
+This rule additionally supports TypeScript type syntax.
+
+Examples of **incorrect** TypeScript code for this rule:
+
+:::incorrect
+
+```ts
+/*eslint no-invalid-this: "error"*/
+
+interface SomeType {
+    prop: string;
+}
+
```

Comment:
```suggestion
```

This interface doesn't seem to be used anywhere in the example, so it might be confusing why it is here?